Packs/SailPointIdentityNow/Integrations/SailPointIdentityNow/SailPointIdentityNow.py

```diff
@@ -0,0 +1,622 @@
+        "clientMetadata": {}
+    }
+    response = client.send_request(IDN_BETA_ACCESS_REQUEST_EXT, "POST", None, json_data)
+    if response is not None and 200 <= response.status_code < 300:
+        return 'Access request was successful!'
+    elif 'detailCode' in response.json():
+        return response.json()
+    return None
+
+
+def access_request(client: Client, request_type: str, requested_for: str, requested_item: str, requested_item_type: str,
+                   comment: str):
+    """
+    Grant or revoke access request for a single object(access profile or role) for a single user.
+    Command(s): identitynow-request-grant, identitynow-request-revoke
+
+    :type client: ``Client``
+    :param client: SailPoint client
+
+    :type request_type: ``str``
+    :param request_type: Type of access request(GRANT_ACCESS or REVOKE_ACCESS).
+
+    :type requested_for: ``str``
+    :param requested_for: Identity Id for whom the access request is being made.
+
+    :type requested_item: ``str``
+    :param requested_item: Id of the object(access profile or role).
+
+    :type requested_item_type: ``str``
+    :param requested_item_type: Type of object(ACCESS_PROFILE or ROLE).
+
+    :type comment: ``str``
+    :param comment: Comments to attach to the item request.
+
+    :return: Success or failure status.
+    """
+    if request_type is None or requested_for is None or requested_item is None or requested_item_type is None:
+        return None
+
+    json_data = {
+        "requestedFor": [requested_for],
+        "requestType": request_type,
+        "requestedItems": [
+            {
+                "type": requested_item_type,
+                "id": requested_item,
+                "comment": comment
+            }
+        ],
+        "clientMetadata": {}
+    }
+    response = client.send_request(IDN_BETA_ACCESS_REQUEST_EXT, "POST", None, json_data)
+    if response is not None and 200 <= response.status_code < 300:
+        return 'Access request was successful!'
+    elif 'detailCode' in response.json():
+        return response.json()
+    return None
+
+
+''' MAIN FUNCTION '''
+
+
+def main():
+    """
+    Intercept and execute commands.
+    """
+
+    # IdentityNow API Base URL (https://org.api.identitynow.com)
+    base_url = demisto.params().get('identitynow_url')
+
+    # OAuth 2.0 Credentials
+    client_id = demisto.params().get('client_id')
+    client_secret = demisto.params().get('client_secret')
+    grant_type = 'client_credentials'
+
+    # Convert the argument to an int or set to MAX_INCIDENTS_TO_FETCH
+    max_results = int(demisto.params().get('max_fetch'))
+    if not max_results or max_results > MAX_INCIDENTS_TO_FETCH:
+        max_results = MAX_INCIDENTS_TO_FETCH
+
+    # first_fetch_str = demisto.params().get('first_fetch', '3 days')
+
+    # Other configs
+    verify_certificate = not demisto.params().get('insecure', False)
+    proxy = demisto.params().get('proxy', False)
+    request_timeout = 10
```

Comment:
```suggestion
    params = demisto.params()
    base_url = params.get('identitynow_url')

    # OAuth 2.0 Credentials
    client_id = params.get('client_id')
    client_secret = params.get('client_secret')
    grant_type = 'client_credentials'

    # Convert the argument to an int or set to MAX_INCIDENTS_TO_FETCH
    max_results = int(params.get('max_fetch'))
    if not max_results or max_results > MAX_INCIDENTS_TO_FETCH:
        max_results = MAX_INCIDENTS_TO_FETCH

    # first_fetch_str = params.get('first_fetch', '3 days')

    # Other configs
    verify_certificate = not params.get('insecure', False)
    proxy = params.get('proxy', False)
    request_timeout = 10
```

Reply:
Is there any advantage of doing - params = demisto.params() vs demisto.params().get(params)?

Reply:
Updated.